src/version-2-1-0.md

```diff
@@ -0,0 +1,31 @@
+Date: 2016-03-09
+Title: Release 2.1.0
+Tags: news, release notes
+Slug: uniqush-push-2-1-0
+Category: news
+
+Please check out our [release notes] for version 2.1.0.
+
+We've added new APIs for listing the subscriptions of a subscriber and for
+listing the services that exist.
```

Comment:
"listing the services that exist" -> "showing the configurations (of the push service providers) for services that exist."

1. Services can have multiple PSPs.
2. It doesn't just list services, it shows all of the information about that service.